### DISCUSSION TITLE
`fish` always on top?

### DISCUSSION THREAD

**Discussion by mangkoran** [fish-shell/fish-shell#8574] [Q&A] [answered]
I am pretty new to `fish`. How do I make `fish` shell prompt always on top similar to https://github.com/swirepe/alwaysontop?

**Comment by faho** [comment, marked as answer]
Fish has no specific support for this. The best you can do is probably something like run `clear` after each command (the `fish_postexec` event could be used for this).

However, this would at the very least move the output above the prompt, which is awkward, and if your terminal or clear want, it will even clear the scrollback buffer (typically you can inhibit that with `clear -x`, but in my tests that doesn't actually work with konsole - it still clears the scrollback so there's no way to see the output).

So: Basically not, and I don't recommend this. If you really want to try, see the fish_postexec event.
